css/src/components/form/textarea.scss

```diff
@@ -0,0 +1,88 @@
+$textarea-color: $text !default;
+$textarea-background-color: $body-background !default;
+$textarea-border-color: $text !default;
+$textarea-placeholder-color: $text-subtle !default;
+
+$textarea-hover-color: $text !default;
+$textarea-hover-border-color: $primary !default;
+$textarea-focus-border-color: $primary !default;
+
+$textarea-disabled-color: $text-subtle !default;
+$textarea-disabled-background-color: $body-background-medium !default;
+$textarea-disabled-border-color: $table-border-dark !default;
+
+$textarea-error-border-color: $danger !default;
+$textarea-warning-border-color: $warning-active !default;
```

Comment:
same comments as related to help, these color concepts need to match more closely with their names

Reply:
updated